lib/vendor/PEAR/Builder.php

```diff
@@ -106,7 +106,7 @@ function _build_win32($descfile, $callback = null)
         $command = 'msdev '.$dsp.' /MAKE "'.$info['package']. ' - Release"';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$info" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)